06-find.md

```diff
@@ -102,25 +102,51 @@ $ grep -n it haiku.txt
 
 Here, we can see that lines 5, 9, and 10 contain the letters "it".
 
-We can combine flags as we do with other Unix commands.
-For example,
-since `-i` makes matching case-insensitive and `-v` inverts the match,
-using them both only prints lines that *don't* match the pattern
-in any mix of upper and lower case:
+We can combine options (i.e. flags) as we do with other Unix commands.
+For example, let's find the lines that contain the word 'the'.
 
 ~~~ {.input}
-$ grep -i -v the haiku.txt
+$ grep -n -w the haiku.txt
 ~~~
 ~~~ {.output}
-You bring fresh toner.
+2:Is not the true Tao, until
+6:and the presence of absence:
+~~~
 
-With searching comes loss
+Now we want to use the option '-i' to make our search case-insensitive:
```

Comment:
"option '-i' to make" -> "option `-i` to make"

We use backtick quotes for flags.